src/VirtualClient/VirtualClient.Main/profiles/PERF-GPU-MLPERF-TRAINING.json

```diff
@@ -0,0 +1,83 @@
+{
+    "Description": "MLPerf Training GPU Performance Workload",
+    "Metadata": {
+        "RecommendedMinimumExecutionTime": "10:00:00",
+        "SupportedPlatforms": "linux-x64",
+        "SupportedOperatingSystems": "Ubuntu",
+        "SpecialRequirements": "This is an Nvidia GPU-specialized workload. It depends upon the system having an Nvidia GPU card/chip."
+    },
+    "Parameters": {
+        "Username": "azureuser",
+        "DiskFilter": "SizeGreaterThan:1000gb",
+        "LinuxCudaVersion": "12.0",
+        "LinuxDriverVersion": "525",
+        "LinuxLocalRunFile": "https://developer.download.nvidia.com/compute/cuda/12.0.0/local_installers/cuda_12.0.0_525.60.13_linux.run"
+    },
+    "Actions": [
+        {
+            "Type": "MLPerfTrainingExecutor",
```

Comment:
Thank you for taking a stab at adding MLPerf training models to VC. One small ask here is that we try to merge the changes you are making with the existing MLPerfExecutor. You should be able to do something such as leverage an additional parameter added (e.g. ModelCategory = Training vs. ModelCategory = Inferencing) so that you can have a safe fork in the existing code.

This will help us keep the consistency of the existing codebase and profiles up to par. Additionally, we really do want to run both inferencing and training models in our existing PERF-GPU-MLPERF.json profile scenarios. We just had not gotten to the implementation.

Reply:
I spoke with Prachi and she indicated it would be good to keep the training model profile separate for now.

Reply:
For profile, yes totally. how about for the code in executor? Should the executor me separated or merged

Reply:
We would ideally have a single MLPerfExecutor that can handle both inferencing as well as training models even if we keep separate profiles. If this is relatively straight-forward in the executor implementation, let's go for it.

Reply:
I would suggest that lets observe the code change required in MLperf for both training and inference as we are moving from A100 to H100, this will give us an idea on what things are dynamic in both the phases and what are constant and what steps are common in both inference and training. Till that time we will have sufficient test runs for both training and inference to be stable and then we can confidently merge both of them with hope that there will be relatively fewer issues in future GPUs. What do you think? Basically, should we merge profiles and executors once we have tested both training and inference on both A100 and H100?